packages/components/src/Filter/FilterDateRangePicker/_docs/FilterDateRangePicker.stickersheet.stories.tsx

```diff
@@ -168,7 +195,7 @@ export const StickerSheetDefault: StickerSheetStory = {
 export const StickerSheetRTL: StickerSheetStory = {
   ...StickerSheetTemplate,
   name: "Sticker Sheet (RTL)",
-  parameters: { textDirection: "rtl" },
+  parameters: { textDirection: "rtl", delay: 800 },
```

Comment:
Is this delay in general, or just for Chromatic (you'll need to change it - assertion would probably be the best)?
https://www.chromatic.com/docs/delay/

Reply:
Mistake, it was to try getting the snapshot to maybe adjust itself after some delay. But didn't even work. Removed